packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -755,11 +725,8 @@ class _DefaultPub implements Pub {
   ///
   /// For more information, see:
   ///   * [generateLocalizations], `in lib/src/localizations/gen_l10n.dart`
-  Future<void> _updatePackageConfig(
-    File packageConfigFile,
-    Directory generatedDirectory,
-    bool generateSyntheticPackage,
```

Comment:
Nit: seems like the doc for this method is outdated now with the reference to the `generateSyntheticPackage` argument